footer.php

```diff
@@ -93,7 +93,7 @@ function draw_navbuttons() {
         if ($mod->sectionnum > 0 && $sectionnum != $mod->sectionnum) {
             $thissection = $sections[$mod->sectionnum];
 
-            if ($thissection->visible || !$COURSE->hiddensections ||
+            if ($thissection->visible || /*!$COURSE->hiddensections ||*/
```

Comment:
If you're removing obsolete code, please delete it, rather than commenting it out.